Mathlib/NumberTheory/SmoothNumbers.lean

```diff
@@ -0,0 +1,155 @@
+/-
+Copyright (c) 2023 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.LinearAlgebra.ExteriorAlgebra.Grading
```

Comment:
```suggestion
```
This import seems to be redundant to me

Reply:
OK, yes. It was suggested by `#minimize_imports`, and I didn't check.